draft-ietf-quic-transport.md

```diff
@@ -2733,6 +2733,9 @@ a small packet might result in Stateless Reset not being useful in detecting
 cases of broken connections where only very small packets are sent; such
 failures might only be detected by other means, such as timers.
 
+An endpoint that wants to improve the chances that it triggers a Stateless Reset
+when its peer loses state can pad all packets it sends to 38 octets.
```

Comment:
All packets seems like a lot of padding.  In practice, I'd only expect people to pad timer-initiated packets like PING for keepalive or TLP/RTO packets.

Reply:
Yes. Mikkel pointed out that if you do that, all short packets are 38 bytes, except for stateless reset which are exactly 37, and therefore stand out. That's not a hugely useful property. What Ian suggests makes more sense, and in that case you want to pad to something unpredictable, for example between 38 and 256. But this is of course a game of smoke and mirrors, so you want the peer to also randomly pad the responses to ping and tlp/rto. Oh well...